src/components/table/LightTable.jsx

```diff
@@ -132,14 +114,14 @@ export class LightTable extends React.PureComponent {
     handleEnumeration(rowsPerPage, offset, orderBy, orderDirection, searchString);
   };
 
-  handleSearchString = (event) => {
-    const { handleEnumeration } = this.props;
-    const { page, orderDirection, orderBy, rowsPerPage } = this.state; // limit
-    const offset = page * rowsPerPage;
-    const searchString = event.target.value;
-    this.setState({ searchString }); // filter
-    handleEnumeration(rowsPerPage, offset, orderBy, orderDirection, searchString);
-  };
+  componentDidUpdate(prevProps) {
+    const { handleEnumeration, searchString } = this.props;
+    if (prevProps.searchString !== searchString) {
```

Comment:
Again, it feels like we should be handling this through an action (not sure if handleEnumeration does that under the covers...I don't have the code handy right now)

Also...and I think it's likely beyond the scope of this PR...I was wondering if we could see about making this debounce to search rather than on every keystroke...that could end up hammering the API

Reply:
I would think that depends on whether we are storing the search string in the redux store, right?

Also, please take a look at the latest commit - this changes w/ the functional component and the useEffect hook so that we no longer need to check if the search string is different.